applications/about.md

```diff
@@ -1,11 +1,9 @@
 # Applications
 
-Bandwidth uses HTTP Callbacks (also known as [webhooks](https://webhooks.pbworks.com/w/page/13385124/FrontPage)) to send [message events](../events/messageEvents.md) to any publicly addressable url.
+Bandwidth uses HTTP Callbacks (also known as [webhooks](https://webhooks.pbworks.com/w/page/13385124/FrontPage)) to send [message events](../messaging/callbacks/messageEvents.md) to any publicly addressable url.
 
 In order to successfully use the Messaging 2.0 APIs, you need to configure an `Application` which contains the `CallbackUrl`.  Each application can be assigned to as many `Locations` as needed.  However, each `Location` can only have a single `Application`
 
-![Application Diagram](../images/messaging-2/applications-flow-messaging.png)
```

Comment:
We'll need an updated graphic from UX

Reply:
Yes. I didn't see a link to an appropriate graphic